ci/acceptance-test/tests/02_start_and_terminate_lxc_instance_test.go

```diff
@@ -35,7 +35,7 @@ func waitUntil(t *testing.T, d time.Duration, cb func()error) {
 }
 
 func TestLXCInstance(t *testing.T) {
-	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc")
+	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc", `{"interfaces":[{"type":"veth", "bridge":"linux"}]}`)
```

Comment:
I've been thinking while using this code. Why does the user have to know what bridge type the instance will get attached to? Isn't it better to just configure that on the executor and let each executor handle it?